packages/contracts-bedrock/contracts/test/SystemConfig.t.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { CommonTest } from "./CommonTest.t.sol";
+import { SystemConfig } from "../L1/SystemConfig.sol";
+
+contract SystemConfig_Init is CommonTest {
+    SystemConfig sysConf;
+
+    function setUp() external {
+        sysConf = new SystemConfig({
+            _owner: alice,
+            _overhead: 2100,
+            _scalar: 1000000,
+            _batcherHash: bytes32(hex"abcd"),
+            _gasLimit: 9_000_000
+        });
+    }
+}
+
+contract SystemConfig_Initialize_TestFail is CommonTest {
+    function test_initialize_lowGasLimit_reverts() external {
+        vm.expectRevert("SystemConfig: gas limit too low");
+        new SystemConfig({
+            _owner: alice,
+            _overhead: 0,
+            _scalar: 0,
+            _batcherHash: bytes32(hex""),
+            _gasLimit: 7_999_999
```

Comment:
I don't love how `7_999_999` is a magic value - it should have a comment as to how this value was picked

Reply:
ok fixed.